sapnwbootstrap-formula.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Aug 27 08:19:27 UTC 2021 - Eike Waldt <waldt@b1-systems.de>
+
+- Version bump 0.6.7
+  * fix non-existing procedure file when running NW import on a HANA scale-out system
```

Comment:
I would split this entry in 2 lines to make the text more compacted as the other entries in the file

Reply:
fixed